.github/workflows/rolling-instance-actions.yml

```diff
@@ -12,6 +12,9 @@ jobs:
     env:
       RUNNER_DEBUG: 1
       WF_USERNAME: davidjen@ppdevtools.onmicrosoft.com
+      WF_SPN_ENV: "https://gintonic30.crm.dynamics.com/"
```

Comment:
I had to  hardcode the environmentUrl for SPN authentication because for a newly created environment, there are couple of manual steps that we need to perform to set it up for SPN authentication.
"who-am-i action with appId-clientSecret" would not work for a newly created environment